app/views/user_mailer/invite.text.erb

```diff
@@ -0,0 +1,11 @@
+Hi!
+
+We're excited to invite you to be one of the earliest users of RapidSkills, a resource to help you find, copy, and customize apprenticeship work process and competency outlines. Create your account now at:
+
+<%= new_registration_url(name: @user.name, email: @user.email, organization: @user.employer.try(:name)) %>
```

Comment:
This won't actually work because an account with their email already exists.  We really need to send them a reset password link.